dist/unix/org.qbittorrent.qBittorrent.appdata.xml

```diff
@@ -72,4 +72,10 @@
   <url type="donation">https://www.qbittorrent.org/donate</url>
   <url type="help">http://forum.qbittorrent.org/</url>
   <url type="translate">https://github.com/qbittorrent/qBittorrent/wiki/How-to-translate-qBittorrent</url>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-location">intense</content_attribute>
```

Comment:
perhaps it should be `mild`? because trackers could do that.

Reply:
The program itself however doesn't leak any location info (think smartphone and gps pos)

Reply:
I was thinking about normal/liberal use case, but I don't insist, removal of this is also fine for me.